app/controller/sdl/Abstract/Controller.js

```diff
@@ -263,6 +263,23 @@ SDL.SDLController = Em.Object.extend(
         SDL.SDLModel.data.mediaPlayerActive = status;
       }
       FFW.BasicCommunication.OnEventChanged(eventName, status);
+      
+      if ('PHONE_CALL' == eventName) {
+        if (true == status) {
+          for (var i = SDL.SDLModel.data.registeredApps.length - 1; i >= 0; i--) {
+            if (SDL.SDLController.getApplicationModel(SDL.SDLModel.data.registeredApps[i].appID).level == 'FULL') {
+              SDL.SDLModel.data.redisplayAppID = SDL.SDLModel.data.registeredApps[i].appID;
+              FFW.BasicCommunication.OnAppDeactivated(SDL.SDLModel.data.redisplayAppID);
+            }
+            break;
+          }
```

Comment:
```suggestion
          if (SDL.SDLController.model) {
             SDL.SDLModel.data.redisplayAppID = SDL.SDLController.model.appID;
             FFW.BasicCommunication.OnAppDeactivated(SDL.SDLModel.data.redisplayAppID);
          }
```
`AppModel.level` isn't really maintained perfectly as far as I can tell, so it probably isn't a reliable way of finding the active app. A much easier way to do this would be to check `SDL.SDLController.model`, since it is keeps a copy of the active app model when available.

Reply:
@jacobkeeler -san,
Sorry for late response.
We are confirming now.
Please wait.